components/constellation/src/lib.rs

```diff
@@ -0,0 +1,170 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
this should be in constellation/rs/cxx/src and the associated BUILD.gn should go in constellation/rs/cxx

Reply:
also it should have restricted visibility to a `group("rust_lib")` target in constellation/rs/BUILD.gn. See components/skus